proposals/csharp-9.0/function-pointers.md

```diff
@@ -358,9 +361,13 @@ the attribute:
 
 * It is an error to directly call a method annotated with this attribute from C#. Users must obtain a function pointer to
 the method and then invoke that pointer.
-* It is an error to apply the attribute to anything other than a static method. The C# compiler will mark any non-static
-methods imported from metadata with this attribute as unsupported by the language.
-* It is an error to have managed types as parameters or the return type of a method marked with the attribute.
+* It is an error to apply the attribute to anything other than an ordinary static method or ordinary static local function.
```

Comment:
Just wondering, but why are operators/properties restricted here? Is it just a language limitation, similar to binding against a signature that takes `explicit this` for instance methods?

Reply:
Correct, this is a language limitation.